src/main/yamcs/etc/yamcs.oresat0.yaml

```diff
@@ -59,6 +62,22 @@ dataLinks:
     port: 10016
     packetPreprocessorClassName: org.oresat.uniclogs.tctm.EdlPacketPreprocessor
 
+  - name: satnogs-tc-realtime
+    class: org.yamcs.tctm.UdpTcDataLink
+    stream: satnogs_tc_realtime
+    host: localhost
+    port: 10017
+    commandPostprocessorClassName: org.oresat.uniclogs.tctm.SatnogsCommandPostprocessor
+    commandPostprocessorArgs:
+      envService: SatnogsTransferService
+
+  - name: satnogs-tm-realtime
+    class: org.yamcs.tctm.UdpTmDataLink
+    stream: satnogs_tm_realtime
+    host: localhost
+    port: 10018
+    packetPreprocessorClassName: org.oresat.uniclogs.tctm.SatnogsPacketPreprocessor
```

Comment:
This can also be axed, as we are not processing a unique TLM frame from satnogs that our MDB recognizes.